sci-physics/fastjet-contrib/fastjet-contrib-1.055.ebuild

```diff
@@ -23,6 +23,7 @@ RDEPEND="${DEPEND}"
 PATCHES=(
 	"${FILESDIR}"/${PN}-1.049-soname.patch
 	"${FILESDIR}"/${PN}-1.049-ar.patch
+	"${FILESDIR}"/${PN}-1.055-ar.patch # https://github.com/fjcontrib/fjcontrib/issues/3
```

Comment:
I think we should rename the second patch (ar-2, ar-part2, something like this). Otherwise it looks a bit confusing.